configure.ac

```diff
@@ -2322,6 +2322,18 @@ fi
         AC_DEFINE([CLS],[64],[L1 cache line size])
     fi
 
+# mscgen for devguide images
+    AC_PATH_PROG([HAVE_MSCGEN], mscgen, "no")
```

Comment:
I do not know if this should be part of this PR, but `disable-docs` would be a nice configure option